### ISSUE TITLE
Изменился способ подключения стороннего драйвера в selenium

### ISSUE THREAD

**Issue by Dimmas** [yandex/YandexDriver#17]
Hi! В readme-файле описан способ выбрать yandexdriver при использовании selenium в Python.
Пример некорректный, т.к. по всей видимости версия selenium обновилась и теперь для работы с yandexdriver актуален следующий код:

from selenium import webdriver
from selenium.webdriver.chrome.service import Service as ChromeService

binary_yandex_driver_file = r'yandexdriver'  # path to YandexDriver

options = webdriver.ChromeOptions() 

service = ChromeService(executable_path=binary_yandex_driver_file)
driver = webdriver.Chrome(service=service)

driver.get('https://yandex.ru')

**Comment by thewesda** [comment]
> Hi! В readme-файле описан способ выбрать yandexdriver при использовании selenium в Python. Пример некорректный, т.к. по всей видимости версия selenium обновилась и теперь для работы с yandexdriver актуален следующий код:
> 
> from selenium import webdriver from selenium.webdriver.chrome.service import Service as ChromeService
> 
> binary_yandex_driver_file = r'yandexdriver' # path to YandexDriver
> 
> options = webdriver.ChromeOptions()
> 
> service = ChromeService(executable_path=binary_yandex_driver_file) driver = webdriver.Chrome(service=service)
> 
> driver.get('https://yandex.ru')

@Dimmas , Димас, помоги, писал дипломную, делал прогу, обновил селениум и понеслись ошибки, пробовал откаты версий, ниче не помогает xD
Использую версию селениума, 4.11.2
Питон новый 3.11.5
использовал чисто твой пример, до этого сам пытался, но была такая же ошибка (в скрине)
![image](https://github.com/yandex/YandexDriver/assets/107184351/5368a154-7128-4517-9c55-b4b81551f518)

**Comment by thewesda** [comment]
@Dimmas Прошу прощения, сам сделал )
binary_yandex_driver_file = r'yandexdriver.exe'
".exe" добавил

**Comment by Dimmas** [comment]
мой пример для Linux

**Comment by thewesda** [comment]
@Dimmas Привет, ты работаешь с веб-таблицами? Я всё, что мог перерыл для веб-таблиц, не могу сделать код. Точнее ошибка, постоянно.

**Comment by Pager-290583** [comment]
@Dimmas Подскажите пожалуйста как на маке юзать драйвер?

Если я все правильно сделал, то не подскажите как права расшарить драйвер?

Message: 'yandexdriver' executable may have wrong permissions.

**Comment by Dimmas** [comment]
> @Dimmas Подскажите пожалуйста как на маке юзать драйвер?
> 
> Если я все правильно сделал, то не подскажите как права расшарить драйвер?
> 
> Message: 'yandexdriver' executable may have wrong permissions.

Не являюсь пользователем МакОС. Судя по ошибке selenium видит драйвер. Попробуйте выполнить в терминале  chmod 755 'path to YandexDriver'

**Comment by Pager-290583** [comment]
> > @Dimmas Подскажите пожалуйста как на маке юзать драйвер?
> > Если я все правильно сделал, то не подскажите как права расшарить драйвер?
> > Message: 'yandexdriver' executable may have wrong permissions.
> 
> Не являюсь пользователем МакОС. Судя по ошибке selenium видит драйвер. Попробуйте выполнить в терминале chmod 755 'path to YandexDriver'

Да, исследовательским путем завел драйвер. Права на папку, потом ещё надо макси разрешить запуск программы от неизвестного разработчика. И все, работает. Спасибо.
